### PR TITLE
id-check-added

### PR DESCRIPTION
**Pull request by Mike-Kirichenko**

### CODE REVIEW

File: controllers/expenses.controller.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ exports.make = (req, res) => {
 exports.delete = (req, res) => {
```
Comment (usatovaDasha) [inline controllers/expenses.controller.js:21]: I don't see a change to the make function

File: controllers/expenses.controller.js [hunk 2]
```diff
@@ -2,13 +2,12 @@ const db = require("../models");
...
 exports.make = (req, res) => {
-  if
-  (
-    !req.body.hasOwnProperty("name") && 
-    !req.body.hasOwnProperty("price")
-  ) {
+
+  const {body} = req;
+
+  if( !body["name"] || !body["price"] ) {
```
Comment (usatovaDasha) [inline controllers/expenses.controller.js:8]: can I only send one field?

Comment (Mike-Kirichenko) [reply controllers/expenses.controller.js:8]: no

Comment (usatovaDasha) [reply controllers/expenses.controller.js:8]: console.log(true||false); or console.log(false || true);

**Pull request merged**